test/qunit/model/FHIRModel.integration.js

```diff
@@ -435,7 +435,7 @@ sap.ui.define([
 		var mParameters = {
 			success : function(oData){
 				var oValidateResponse = TestUtils.loadJSONFile("Validate");
-				assert.deepEqual(oData, oValidateResponse, "Validate response was correct");
```

Comment:
We should check here following things:

- the whole response is identical with the expected response only the resourceId is different
- the resourceId is has valid UUIDv4 value

Reply:
done

Reply:
done